### DISCUSSION TITLE
message_db prefix not getting used

### DISCUSSION THREAD

**Discussion by jshapiro2** [postalserver/postal#3216] [Development discussions]
Hello,

I have the message_db prefix set to "postalsb" in the postal.yml file.  I just noticed that message_db being created are starting with postal- and not postalsb.  Did I do something wrong?

**Comment by willpower232** [comment]
Did you restart Postal after changing the config file?

**Comment by jshapiro2** [comment]
Yes, same config from when I built this server and just noticed the issue.
